lib/conduit_async_ssl.ml

```diff
@@ -50,4 +50,9 @@ let ssl_listen ~crt_file ~key_file rd wr =
   >>= fun app_rd ->
```

Comment:
As the core committer, I'm not much more help -- the Async_ssl interface is terribly documented.  Am reading the source code now...